exasol_sphinx_github_pages_generator/deploy_github_pages.py

```diff
@@ -79,7 +74,9 @@ def build_and_copy_documentation(build_dir, worktree, source_branch, source_dir,
     # the package found in "module_path"
     # -T: not table of contents
     # -e: put documentation for ech module on own page
-    run(["sphinx-apidoc", "-T", "-e", "-o", "api", module_path], check=True)
+    for module in module_path:
```

Comment:
Seems the type of module_path has changed. It would be nice to have type hints on the method signature...

Reply:
I will add those in the documentation ticket.